dev-vcs/git-cola/git-cola-2.6.ebuild

```diff
@@ -7,7 +7,7 @@ EAPI=5
 PYTHON_COMPAT=( python2_7 python3_{3,4} )
 DISTUTILS_SINGLE_IMPL=true
 
-inherit distutils-r1 readme.gentoo virtualx
+inherit distutils-r1 readme.gentoo-r1 virtualx
```

Comment:
readme.gentoo-r1 requires explicit pkg_postinst. Thus transition to readme.gentoo-r1 is incomplete.